gcc -march=i686 -O2 -pipe -D_FILE_OFFSET_BITS=64 -D_REVISION=0 src/airtun-ng.c src/common.c src/crc.c src/crypto.c -o airtun-ng
In file included from src/airtun-ng.c:35:
/usr/include/linux/wireless.h:885: error: 'IFNAMSIZ' undeclared here (not in a function)
diff -Naur aircrack-ng-0.7.orig/src/airtun-ng.c aircrack-ng-0.7/src/airtun-ng.c
--- aircrack-ng-0.7.orig/src/airtun-ng.c	2007-01-08 19:07:40.000000000 +0100
+++ aircrack-ng-0.7/src/airtun-ng.c	2007-01-22 18:45:47.000000000 +0100
@@ -32,6 +32,7 @@
 
 #include <netpacket/packet.h>
 #include <linux/if_ether.h>
+#include <linux/if.h>
 #include <linux/wireless.h>
 #include <arpa/inet.h>
 #include <unistd.h>
